src/App/Abstractions/IPushNotificationService.cs

```diff
@@ -9,5 +10,7 @@ public interface IPushNotificationService
         Task<string> GetTokenAsync();
         Task RegisterAsync();
         Task UnregisterAsync();
+        void SendLocalNotification(string title, string message, string notificationId = null);
```

Comment:
I think we could avoid having a `null` default value on `notificationId`. For now at least is not being used and I see that you're generating a random number for the id in the implementations, the problem with that is that the id you generate won't be known by the caller so the caller won't be able to dismiss the notification.
So for now I think we don't need the `null`, and if it's null or empty in the implementation you can just throw an `ArgumentNullException`. If in the future we need this we should change the method to return the newly created `notificationId`